src/Actions/MakeQueueTenantAwareAction.php

```diff
@@ -88,17 +89,14 @@ protected function findTenant(JobProcessing $event): Tenant
 
     protected function getJobFromQueueable(object $queueable)
     {
-        switch (get_class($queueable)) {
-            case SendQueuedMailable::class:
-                return $queueable->mailable;
-            case SendQueuedNotifications::class:
-                return $queueable->notification;
-            case CallQueuedListener::class:
-                return $queueable->class;
-            case BroadcastEvent::class:
-                return $queueable->event;
-            default:
-                return $queueable;
+        $job = Arr::get(config('multitenancy.queueable_to_job'), get_class($queueable));
+
+        if (! $job) {
+            return $queueable;
+        } elseif (method_exists($queueable, $job)) {
```

Comment:
Do we need `else` here? Couldn't it be a regular `if`?

Reply:
Ok ;)